jaxlib/setup.py

```diff
@@ -25,19 +24,25 @@
 if cuda_version:
   __version__ += "+cuda" + cuda_version.replace(".", "")
 
-binary_libs = [os.path.basename(f) for f in glob('jaxlib/*.so*')]
-binary_libs += [os.path.basename(f) for f in glob('jaxlib/*.pyd*')]
+# This file is required by PEP-561. It marks jaxlib as package containing
+# type stubs.
+with open('jaxlib/py.typed', 'w') as f:
```

Comment:
I'd be tempted to do this as part of the build_wheel script, just to keep all the package assembly code together.

Reply:
Done.